runtime/src/lib.rs

```diff
@@ -312,7 +328,7 @@ impl vln_human_swap::Config for Runtime {
 
 impl vln_transfers::Config for Runtime {
     type Event = Event;
-    type Assets = Tokens;
+    type Assets = Collateral;
```

Comment:
I think this would still be be `Tokens`(where USDv is stored) as Collateral wont be transferred with this pallet, for now collateral stays with the person that minted/`attest`ed it.

Reply:
Similarly the `Assets` of foreign-asset would rather be `Collateral` as we will `attest` things like `USDC` held externally